tests/encryption_/tests.py

```diff
@@ -0,0 +1,74 @@
+import json
+import sys
+from io import StringIO
+
+from django.core.management import call_command
+from django.db import connections
+from django.test import TestCase, modify_settings, override_settings
+
+from .models import Patient, PatientRecord
+from .routers import TestEncryptedRouter
+
+EXPECTED_ENCRYPTED_FIELDS_MAP = {
+    "billing": {
+        "fields": [
+            {"bsonType": "string", "path": "cc_type", "queries": {"queryType": "equality"}},
+            {"bsonType": "int", "path": "cc_number", "queries": {"queryType": "equality"}},
+        ]
+    },
+    "patient_record": {
+        "fields": [{"bsonType": "string", "path": "ssn", "queries": {"queryType": "equality"}}]
+    },
+    "patient": {
+        "fields": [
+            {"bsonType": "int", "path": "patient_id"},
+            {"bsonType": "int", "path": "patient_age", "queries": {"queryType": "range"}},
+            {"bsonType": "string", "path": "patient_name"},
+        ]
+    },
+}
+
+
+@modify_settings(
+    INSTALLED_APPS={"prepend": "django_mongodb_backend"},
+)
+@override_settings(DATABASE_ROUTERS=[TestEncryptedRouter()])
+class EncryptedModelTests(TestCase):
+    databases = {"default", "encrypted"}
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.patient_record = PatientRecord(ssn="123-45-6789")
+        cls.patient_record.save()
+        cls.patient = Patient(patient_id=1, patient_age=47)
+        cls.patient.save()
+
+    def test_get_encrypted_fields_map_method(self):
+        self.maxDiff = None
+        with connections["encrypted"].schema_editor() as editor:
+            collection_name = self.patient._meta.db_table
+            self.assertCountEqual(
+                {"fields": editor._get_encrypted_fields_map(self.patient)},
+                EXPECTED_ENCRYPTED_FIELDS_MAP[collection_name],
+            )
+
+    def test_get_encrypted_fields_map_command(self):
+        class Tee(StringIO):
+            """Print the output of management commands to stdout."""
+
+            def write(self, txt):
+                sys.stdout.write(txt)
+                super().write(txt)
+
+        out = Tee()
```

Comment:
`Tee` is for temporary debugging?

Reply:
`Tee` is because I want to use the output to update `EXPECTED_ENCRYPTED_FIELDS_MAP` and I got tired of adding and removing `print` statements, so, I'd like to leave it in if possible.

Reply:
You can leave it until the PR is ready to merge but we don't any test routinely writing to stdout.